src/main/webapp/app/overview/courses.component.ts

```diff
@@ -71,6 +72,18 @@ export class CoursesComponent implements OnInit, OnDestroy {
                 // TODO: Stephan Krusche: this is deactivate at the moment, I think we need a more generic solution in more components, e.g. using the the existing notification
                 // sent to the client, when a quiz starts. This should slide in from the side.
                 // this.subscribeForQuizStartForCourses();
+
+                // get all exams of courses
+                this.courses.forEach((course) => {
+                    if (course.exams) {
+                        // set course for exam as it is not loaded within the server call
+                        course.exams.forEach((exam) => {
+                            exam.course = course;
+                            this.exams.push(exam);
+                        });
+                    }
+                });
+                this.nextRelevantExams = this.exams.filter((exam) => moment(exam.endDate!).isAfter(moment()));
```

Comment:
Discussed this with @MaisiKoleni . You should use the server time here and not `moment()` and calculate the individual endDate because some students have extra time

Reply:
@stefanwaldhauser I am using the server time now, good catch! 👍 
About calculating the `individual endDate` for each Exam and displaying it for so long, I discussed this with @krusche and it is okay to just display it until the regular `endDate`